.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,44 @@
+name: Integration Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
How are we specifying that this check is non-blocking to PRs? If that has to be done later/separately, that's cool, just wondering if I missed something.

Reply:
PR status checks can be configured on the branch rules of the repo, under `Settings` page, there you can indicate which ones are required. For this repo you can see it in all PRs:

![image](https://user-images.githubusercontent.com/35276119/93503658-021b6780-f8de-11ea-9f66-5b9b3c5462bf.png)

Also I see the following relevant rules of Mergify bot:

![image](https://user-images.githubusercontent.com/35276119/93503690-12cbdd80-f8de-11ea-980a-8e84318904c3.png)

I guess those refer to the workflow job name, which is `build` for the PR build workflow and `it-test` for the integration tests workflow.